packages/homepage-kpis/PageContentWrapper.tsx

```diff
@@ -0,0 +1,21 @@
+import { Box, BoxProps } from '@chakra-ui/react';
+import * as React from 'react';
+
+export type PageContentWrapperProps = React.PropsWithChildren<{
+   className?: string;
+}>;
+
+export function PageContentWrapper(props: BoxProps) {
+   return (
+      <Box
+         w={{ base: 'full', lg: '960px', xl: '1100px' }}
+         mx="auto"
+         px={{
+            base: 0,
+            sm: 6,
+            lg: 0,
+         }}
+         {...props}
+      />
+   );
+}
```

Comment:
There is already a `PageContentWrapper` exported by the `ui` package

```suggestion
```

Reply:
I think dropping it in the UI package is a great idea, especially if it already has the `PageContentWrapper`. I'll move it over now